python/ray/_private/ray_constants.py

```diff
@@ -422,6 +422,17 @@ def env_set_by_user(key):
 
 LANGUAGE_WORKER_TYPES = ["python", "java", "cpp"]
 
+
+# Accelerator constants
```

Comment:
Remove the changes in this file. We recently decide to move all accelerator-specific constants from `ray_constants.py` to `xxx_xpu.py`.